### ISSUE TITLE
High velocity on move_and_slide causes OS freeze

### ISSUE THREAD

**Issue by thegamerx1** [godotengine/godot#41480]
**Godot version:**
3.2.2

**OS/device including version:**
Windows 10
OpenGL ES 3.0 Renderer: GeForce GTX 1650 SUPER/PCIe/SSE2

**Issue description:**
OS freezes after the velocity of the player gets big
![image](https://user-images.githubusercontent.com/34986564/90991214-0088a900-e597-11ea-83ff-cf99eb427366.png)

**Steps to reproduce:**
Open the project below and run it (i would press esc before everything gets slow or you will have to force shutdown your pc)

**Minimal reproduction project:**

[Project.zip](https://github.com/godotengine/godot/files/5115170/Project.zip)

**Comment by Meriipu** [comment]
How much memory do you have? It uses 9.6GiB for me on Linux (but despite running at max CPU does not appear to crash or freeze beyond printing a lot of 
```
ERROR: instance_set_transform: Condition "Math::is_nan(v.x)" is true.
   At: servers/visual/visual_server_scene.cpp:601.
(0, -nan, 0)
```

**Comment by lawnjelly** [comment]
Fixed by #41123.

**Comment by thegamerx1** [comment]
> 
> 
> How much memory do you have? It uses 9.6GiB for me on Linux, but despite running at max CPU does not appear to crash or freeze beyond printing a lot of:
> 
> ```
> ERROR: instance_set_transform: Condition "Math::is_nan(v.x)" is true.
>    At: servers/visual/visual_server_scene.cpp:601.
> (0, -nan, 0)
> ```

I have 8GB of ram I get different errors on console
![image](https://user-images.githubusercontent.com/34986564/91043290-e08ecf00-e602-11ea-876d-7f5b3a7f8893.png)

**Comment by akien-mga** [comment]
Fixed by #41123.
